debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -138,11 +138,13 @@
             PDB_NAME,
             XSTREAM_SERVER_NAME,
             SNAPSHOT_MODE,
-            HistorizedRelationalDatabaseConnectorConfig.DATABASE_HISTORY,
-            RelationalDatabaseConnectorConfig.TABLE_WHITELIST,
-            RelationalDatabaseConnectorConfig.TABLE_BLACKLIST,
-            RelationalDatabaseConnectorConfig.TABLE_IGNORE_BUILTIN,
-            RelationalDatabaseConnectorConfig.MSG_KEY_COLUMNS,
+            OracleConnectorConfig.DATABASE_HISTORY,
+            OracleConnectorConfig.TABLE_WHITELIST,
+            OracleConnectorConfig.TABLE_INCLUDE_LIST,
+            OracleConnectorConfig.TABLE_BLACKLIST,
+            OracleConnectorConfig.TABLE_EXCLUDE_LIST,
+            OracleConnectorConfig.TABLE_IGNORE_BUILTIN,
+            OracleConnectorConfig.MSG_KEY_COLUMNS,
```

Comment:
Beside adding the new include/exclude list properties I did some housekeeping here to move the parameters to the ConnectorConfig class related to that connector in order to local rewrites work and from architectural perspective we have less dependencies to "the outside".

(also in several other cases)

Reply:
WDYM by "moving" exactly? Just referencing those fileds via `OracleConnectorConfig` instead of say `HistorizedRelationalDatabaseConnectorConfig`? If so, I don't think it's a strong improvement; in fact I'd even expect a warning as static members should always be accessed via the defining type instead of via sub-class. I'd prefer to revert this change.

Reply:
I am not such a Java expert like you are because I didn't do professional Java for so many years like you, but I think what you mean is that you don't access a static by an instance, like:
```
class A {
    static int x = 2;
    ...
}

A a = new A();
a.x;
```
Do you have a link that explains it, because neither IDE nor compilation showed an error/warning so I guessed it's valid?

I think it's valid, especially when you mean to override parent properties in `OracleConnectorConfig` you expect the local values to be used. I thought this is more like a super/this replacement in a static scope?

Reply:
You are right, it doesn't generate an error. Note you cannot override a static member though, rather it will be _hidden_ when declaring an identical member in a sub-class.

That said, I preferred the original way of referencing those props via the defining (super-)class. If nothing else, it keeps the code more generic, e.g. making it easier to extract and reuse (or just copy and paste), if needed.

Reply:
But using `RelationalDatabaseConnectorConfig.SERVER_NAME` will always returns the definition from `RelationalDatabaseConnectorConfig`, correct?
And when I use `RelationalDatabaseConnectorConfig.SERVER_NAME` everywhere inside the Oracle connector module I have a dependency everywhere to that external resource. Wouldn't it be much easier to have only the dependency to `OracleConnectorConfig` which resided in the module and handle everything there?
Futhermore I believe it makes it more predictable when I override (also it technically only hides the original value) something in `OracleConnectorConfig` that it is automatically used everywhere and I don't have to switch it on all occasions manually? Gives the connector config a better meaning too?

Reply:
> Wouldn't it be much easier to have only the dependency to OracleConnectorConfig

But that's the thing, the nature of the dependency doesn't change a bit. The code still - rightly so - depends on the property definition from the super-class, it only gets swepped under the rug, if you will. I strongly prefer to rever this change, in any case it's unrelated to the matter at hand. Thanks!

Reply:
Will be reverted

Reply:
Done